src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$days-week:
+  "mon" 0, "tue" 103px, "wed" 206px, "thu" 309px, "fri" 412px,
+  "sat" 515px, "sun" 618px;
+$length-of-month: 29;

```

Comment:
There should be something like that. Its not the best design to hardcode px for each $days-week:
`$cell-size: 100px;
$month-length: 31;
$days-in-a-week: 7;
$gap: 1px;
$padding: 10px;
$max-width: $days-in-a-week * $cell-size + 2 
  *$padding + $gap * ($days-in-a-week - 1);
$week:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;`